mistralrs-web-chat/static/js/utils.js

```
@@ -0,0 +1,107 @@
+// Utility functions for markdown rendering, DOM manipulation, etc.
+
+/**
+ * Render markdown to HTML with proper escaping
+ */
+function renderMarkdown(src) {
+  // Helper: escape & < > once
+  const escape = s =>
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)